### DISCUSSION TITLE
CTRL-ALT-DEL: after installing RUSTDESK, C-A-D doesn't work for other software

### DISCUSSION THREAD

**Discussion by kuerbis42** [rustdesk/rustdesk#5310] [Q&A]
Hi,
it's only a question, I don't know, if it's a bug.
I'm using RADMIN-Software and RUSTDESK - for tesating I installed RUSTDESK on my server next to RADMIN.
(Sorry must write it as code, cause otherwise it is strike-out)
```
Normally with RADMIN, you can use < CTRL > - <ALT> - <F12> to send < CTRL > - <ALT> - <DEL> to the remote PC.
This works well until installing RUSTDESK.
After that, nor < CTRL > - <ALT> - < DEL> neither < CTRL > - <ALT> - <F12> works anymore with RADMIN.
After deinstalling RUSTDESK it doesn't came to work again.
```

Any ideas?
TIA
Frank

**Comment by smsabet** [comment]
Enable Software Secure Attention Sequence (SAS):
1. Open the Local Group Policy Editor on the agent machine:
    Press WIN+R to open the run dialog and type gpedit.msc and press Enter.
2. Expand Computer Configuration > Administrative Templates > Windows Components
3. Select Windows Logon Options.
4. Double-click Disable or enable software Secure Attention Sequence.
5. Select Enabled.
6. Select "Ease of Access applications" from the drop down list in the bottom left pane.
7. Click OK.

**Comment by rustdesk** [comment]
https://admx.help/?Category=Windows_10_2016&Policy=Microsoft.Policies.WindowsLogon2::SoftwareSASGeneration

https://github.com/rustdesk/rustdesk/blob/96edca8f74e195896db9f9929d2aed3100eac47f/src/platform/windows.rs#L1133

https://github.com/rustdesk/rustdesk/blob/96edca8f74e195896db9f9929d2aed3100eac47f/res/msi/CustomActions/CustomActions.cpp#L603

**Reply by rustdesk** [reply]
@fufesou let's enhance this, if v0 = 0, change to 1, if v0 = 2, change to 3, do nothing for the other value, so `v1 = v0 | 1`. Low priority. Please take note.

**Reply by c-michail** [reply]
Actually you need to modify the registry key and use SendSAS as i'm describing here https://github.com/rustdesk/rustdesk/discussions/8962

At least the server edition needs the above 2 methods. Create the registry value if it doesn't exists and after use SendSAS.
You can skip the registry key on Client editions of windows.
